tests/test_hasheous.py

```
@@ -0,0 +1,1559 @@
+
+
+class _Resp:
+    """A minimal stand-in for ``http.client.HTTPResponse``.
+
+    ``read1`` drains and then returns b"" the way a real response does. An
+    earlier version returned the whole payload on every call, which never hit
+    EOF -- so tests naming the JSON-parse path actually passed via the
+    size-limit path instead.
+    """
+
+    def __init__(self, payload: bytes):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)